src/main/java/sg/ncl/ConnectionProperties.java

```diff
@@ -147,6 +147,11 @@ public void setApproveJoinRequest(String approveJoinRequest) {
         this.approveJoinRequest = approveJoinRequest;
     }
 
+    public String getRejectJoinRequest(String teamId, String userId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)